lang/cs/Org.Apache.REEF.Bridge.Client/GrpcGeneratedCode/DriverCommonProtocol.cs

```diff
@@ -0,0 +1,342 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Do we need to check in the generated sources or can we make the codegen part of the build instead?

Reply:
I set this up exactly as we were doing for Avro where we check in the generated sources. The gRPC .NET examples do the same. I'll have to give this some consideration.

Reply:
Maven of course has the proper behavior for code generation and automatically puts the files into the target directory and includes them in the build. By putting the generated files under the project directory, msbuild automatically finds them and builds them which makes adding new .proto files trivial. If we put the generated files outside of the project tree, adding a new .proto file requires telling msbuild what files are generated from a given .proto file and where to find them to add them to build. I believe the first solution is preferable from a usability perspective.